src/comparator.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+  <title>Elastic Maps Service Style Compare</title>
+  <style>
+    h2 {
+      z-index: 9999;
+    }
+    .drag {
+      opacity: 0.5;
+      transition: opacity 0.15s ease-in;
+    }
+    .mapboxgl-ctrl-attrib {
+      display: none;
+    }
+  </style>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="https://unpkg.com/tachyons/css/tachyons.min.css" />
+  <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/v1.11.0/mapbox-gl.css" />
+  <link
+    rel="stylesheet"
+    type="text/css"
+    href="https://unpkg.com/dialog-polyfill@0.5.1/dist/dialog-polyfill.css"
+  />
+  <body>
+    <dialog class="w-50">
```

Comment:
```suggestion
    <dialog class="w-50 dn sans-serif">
```

I'd like the fonts to be consistent with the titles and adding `dn` by default the modal is hidden, a small change later will display the modal only if the `style` param is not present, that way for the first load the modal is shown, but if the user switches styles using the links in the title, or just comes to the comparator from a bookmark with the style, the modal is also omitted.

I guess this is good enough instead of storing a checkmark in the localStorage.